### PR TITLE
Gui 2852 Create services that drive UI

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2852

### CODE REVIEW

File: eucaconsole/templates/reporting/reporting_preferences.pt [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+                    </select>
+                    <small class="error" i18n:translate="">Bucket name is required</small>
+                    <div id="create-bucket-link">
+                        <span i18n:translate="">OR:</span>
+                        <a ng-click="preferences.showCreateBucket()" i18n:translate="">Create bucket</a>
+                    </div>
+                </div>
             </div>
+            <p i18n:translate="">
+                Note: by enabling usage reporting for this bucket, we will update the bucket policy to allow eucalyptus to write reports there.
```
Comment (kamalgill) [inline eucaconsole/templates/reporting/reporting_preferences.pt]: This text doesn't match the wireframe (which was recently updated)

File: eucaconsole/templates/reporting/reporting_preferences.pt [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
-        <p i18n:translate="">
-            User-assigned tag keys have the prefix "user", such as "user:Application". Eucalyptus-assigned tag keys have the prefix "euca". Active tag keys are column headings (e.g. user:tagkey1), while tag values will display as the column values under the tag key heading.
-        </p>
-        <div class="active-tags">
-            <form name="tagKeyForm" ng-submit="preferences.addTagKey()" novalidate="">
+        <hr/>
+        <div class="section">
+            <h6 class="section-label" i18n:translate="">Manage Report Tags</h6>
+            <p i18n:translate="">
+                You can use cost allocation tags to categorize and track your Eucalyptus usage. When you apply tags to your resources (such as EC2 instances or S3 buckets) and activate the tags, Eucalyptus generates a cost allocation report as a comma-separated value (CSV file) with your usage and costs aggregated by your active tag keys. You can apply tags that represent business categories (such as application names or owners) to organize your costs across multiple services.
```
Comment (kamalgill) [inline eucaconsole/templates/reporting/reporting_preferences.pt]: Feel free to add line breaks here and elsewhere, making long paragraphs easier to review (and update).

File: eucaconsole/templates/reporting/reporting_preferences.pt (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/templates/reporting/reporting_preferences.pt]: I didn't see that update, so let me review to be sure all information is captured.

**Comment by dkavanagh** [comment]
@kamalgill This should be good to go. I'll update the next PR to include changes in here.

**Comment by dkavanagh** [comment]
This content is also in a subsequent PR, we'll just merge that one.

**Pull request closed without merging**